packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -385,13 +391,19 @@ export const prepareSettler = (
          *
          * @param {EvmHash} txHash
          * @param {NatValue} fullValue
+         * @param {CosmosChainAddress['value'] | AccountId} EUD
          */
-        async disburse(txHash: EvmHash, fullValue: NatValue) {
+        async disburse(
+          txHash: EvmHash,
+          fullValue: NatValue,
+          EUD: CosmosChainAddress['value'] | AccountId,
+        ) {
           const { repayer, settlementAccount } = this.state;
           const received = AmountMath.make(USDC, fullValue);
           const { zcfSeat: settlingSeat } = zcf.makeEmptySeatKit();
           const { calculateSplit } = makeFeeTools(feeConfig, customFeeConfigs);
-          const split = calculateSplit(received);
+          const accountId = chainHub.resolveAccountId(EUD);
```

Comment:
can this throw? I'm too rushed to think thru whether we're ok if it does.
I wonder about a test.

Reply:
Theoretically, yes. But it shouldn't since we already performed this check in Advancer. Added a comment